docs/users/index.md

```diff
@@ -21,3 +21,72 @@ jupyter labextension list
+
+### Creating a job or job definition
+
+1. Choose a Jupyter notebook
+   - To create a *job* or *job definition* from a file browser, right-click on a notebook in the file browser and choose “Create Notebook Job” from the context menu:
+![“Create Notebook Job” button in the file browser context menu](./images/create_job_from_filebrowser.png)
+
+   - To create a *job* or *job definition* from an open Notebook, click on a “Create a notebook job” button in the top toolbar of the open Notebook
+![“Create a notebook job” button in the top toolbar of the open Notebook](./images/create_job_from_notebook.png)
+
+2. Fill out Create Job form
+
+   You can give your notebook job or job definition a name, choose an environment to run it in, select different output formats, and provide parameters that are set as local variables when your notebook gets executed. This parameterized execution is similar to [Papermill](https://papermill.readthedocs.io/en/latest/); Jupyter Scheduler provides a UI for managing and accessing the outputs of your notebook jobs inside JupyterLab.
+
+   - To create a *job*, press "Create" button when "Run Now" is chosen in "Schedule" section.
+   !["Create Job Form"](./images/create_job_form.png)
+
+   - To create a *job definition* that would run your job on a recurring schedule select “Run on a schedule” radio button in "Schedule" section. We provide a few useful shortcuts to, for example, run your notebook at a specified minute, hour, daily or weekly interval.
+   !["Run on schedule"](./images/run_on_schedule.png)
+   You can also specify a custom schedule in [crontab format](https://www.man7.org/linux/man-pages/man5/crontab.5.html) by selecting "Custom schedule" in "Interval" drop-down menu.
+   !["Custom schedule option"](./images/custom_schedule.png)
+
+### Using list view
+
+Once you’ve created a job or job schedule, you can use the “Notebook Jobs” icon on the launcher to view lists of jobs and job definitions.
+
+!["Launcher with Notebook Jobs icon in Other section"](./images/launcher.png)
+
+Each items name links to a detail view.
+
+!["Job name as a link in each column"](./images/item_name.png)
+
+You can sort the list by clicking one of the headers (for example, click "Job name" header to sort the list alphabetically based on "Job Name").
+
+!["Notebook Jobs list sorted alphabetically after Job Name header is clicked"](./images/headers.png)
+
+You can also delete each job or pause its execution in the list by clicking appropriate buttons in "Actions" column.
+
+!["Delete and Stop buttons in Actions column"](./images/actions_list.png)
+
+When one of your jobs has completed, you can download its files by clicking "Download job files" button in "Ouput files" column.
+
+!["Download button in Output Files column"](./images/download_button.png)
+
+After download is finished, you can access the downloaded files through links that appear instead of "Download job files" button in "Output files" column.
```

Comment:
```suggestion
After clicking the download button, links to files appear in the "Input file" and "Output files" columns.
```

You could also revise `downloaded_files.png` to include the "Input file" column, where a link appears